projects/cloud/app/services/invitation_accept_service.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+class InvitationAcceptService < ApplicationService
+  module Error
+    class Unauthorized < CloudError
```

Comment:
Is this really what we want to do? I will then no longer be able to do `assert_raises(InvitationAcceptService::Error::Unauthorized)` since `assert_raises` assumes the parameter will be a class or a module but in your case it'd be a class instance, afaict at least.